### PR TITLE
libosinfo 1.5.0

### PR DESCRIPTION
**Pull request by alebcay**
Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by alebcay** [comment]
```
make[3]: ../build-aux/install-sh: No such file or directory
make[3]: *** [Libosinfo-1.0.gir] Error 1
make[2]: *** [all] Error 2
make[1]: *** [all-recursive] Error 1
make: *** [all] Error 2
```

Manually verified that `build-aux/install-sh` is present, so not sure why that's failing.

**Comment by jeffreywildman** [comment]
Run `brew install -vd libosinfo` and enter into a shell after the build fails.
Inside the shell, run `install-sh` directly:

```
install-sh: /usr/bin/sh: bad interpreter: No such file or directory
```

The shebang in `install-sh` is hardcoded to look for `sh` in `/usr/bin` instead of `/bin/sh`.
Might be worth poking upstream; also, one workaround may be to invoke `autoreconf` to see if a locally generated `install-sh` works.

File: Formula/libosinfo.rb [hunk 1]
```diff
@@ -27,6 +27,7 @@ def install
       --localstatedir=#{var}
       --mandir=#{man}
       --sysconfdir=#{etc}
+      --disable-dependency-tracking
       --disable-silent-rules
       --disable-udev
```
Comment (jeffreywildman) [inline Formula/libosinfo.rb]: `--disable-udev` can be dropped, it's no longer supported.

https://gitlab.com/libosinfo/libosinfo/commit/16cf1f797728dff9e361033937faea9f38ffbe49

**Comment by alebcay** [comment]
Tested with `autoreconf -ivf` (adding necessary build dependencies); shebang path remains hardcoded. For now, it's fixed with an `inreplace`. Reported issue upstream [here](https://gitlab.com/libosinfo/libosinfo/issues/31).

**Comment by alebcay** [comment]
Upstream has acknowledged the issue, tracing it back to [an `automake` bug](https://bugzilla.redhat.com/show_bug.cgi?id=1709078). For this particular release, we'll have to patch the file ourselves but should be resolved in future releases.

**Comment by SMillerDev** [comment]
@brewtestbot test this please!

**Pull request closed without merging**